config.js

```diff
@@ -18,6 +18,15 @@ module.exports = {
     port: process.env.REDIS_PORT || '6379',
     host: process.env.REDIS_HOST || '127.0.0.1'
   },
+  postgre: {
+    user: process.env.POSTGRE_USER || 'test',
+    host: process.env.POSTGRE_HOST || 'localhost',
+    password: process.env.POSTGRE_PASSWORD || 'test',
+    database: {
+      reports: process.env.POSTGRE_DB_NAME || 'postgres'
```

Comment:
need to change this from postgres to reports or what every name you think the db should be called